tuf/api/metadata.py

```diff
@@ -497,11 +499,15 @@ def __init__(
             raise ValueError(f"Unsupported spec_version {spec_version}")
 
         self.spec_version = spec_version
-        self.expires = expires
 
-        if version <= 0:
+        self.expires = expires or datetime.utcnow() + timedelta(days=1)
+
+        if version is None:
+            version = 1
```

Comment:
The same question about the docstring and default values applies to all attributes.
Do we want to mention the default values?